### ISSUE TITLE
Off-canvas 'close' icon is hidden behind admin bar when users are logged in

### ISSUE THREAD

**Issue by bacoords** [understrap/understrap#1630]
### The Bug

**Describe the bug**
Off-canvas 'close' icon is hidden behind admin bar when users are logged in

**To Reproduce**
Use the Bootstrap5 version and the oddcanvas style header bar. Then view the menu as a logged in WordPress user. The close icon is hidden

**Options**
1. Increase the z-index so that the menu appears above the admin bar
2. Add css to push the off canvas down.. this would need to be tested, but something like this:

```scss
body.admin-bar .offcanvas-end{
  margin-top:46px;
}
```

**Comment by kirannasim** [comment]
It works well, but it also affects the main menu on the desktop screen. so I added the media query as follows.

https://github.com/understrap/understrap/pull/1644

```
@include media-breakpoint-down(md) {
	body.admin-bar .offcanvas-end {
		margin-top: 46px;
	}
}
```

**Comment by IanDelMar** [comment]
@bacoords fixed in v1.2.0
